src/test/kotlin/uk/gov/dluhc/printapi/database/repository/AnonymousElectorDocumentRepositoryIntegrationTest.kt

```diff
@@ -0,0 +1,70 @@
+package uk.gov.dluhc.printapi.database.repository
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import uk.gov.dluhc.printapi.config.IntegrationTest
+import uk.gov.dluhc.printapi.database.entity.SourceType.VOTER_CARD
+import uk.gov.dluhc.printapi.database.repository.AnonymousElectorDocumentRepositoryExtensions.findPendingRemovalOfFinalRetentionData
+import uk.gov.dluhc.printapi.database.repository.AnonymousElectorDocumentRepositoryExtensions.findPendingRemovalOfInitialRetentionData
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildAnonymousElectorDocument
+import java.time.LocalDate
+
+internal class AnonymousElectorDocumentRepositoryIntegrationTest : IntegrationTest() {
```

Comment:
I'm not a massive fan of having repository tests when we can test this functionality higher up the stack, so I may consider removing this in a future PR.